pay-api/src/pay_api/models/invoice.py

```diff
@@ -258,6 +260,7 @@ class Meta(BaseSchema.Meta):  # pylint: disable=too-few-public-methods
 
     # pylint: disable=no-member
     payment_line_items = ma.Nested(PaymentLineItemSchema, many=True, data_key="line_items")
+    partial_refunds = ma.Nested(RefundsPartialSchema, many=True, data_key="partial_refunds")
```

Comment:
This is going to change the payload probably for CSO which is going to break their code